sysdig/resource_sysdig_secure_cloud_auth_account_feature_test.go

```diff
@@ -0,0 +1,181 @@
+//go:build tf_acc_sysdig_secure || tf_acc_sysdig_common
+
+package sysdig_test
+
+import (
+	"fmt"
+	"os"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/acctest"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+
+	"github.com/draios/terraform-provider-sysdig/sysdig"
+)
+
+// TF acceptance tests for secure account feature need an actual azure account
+// onboarded to use its account_id (uuid based) as input to the account feature CRUD calls.
+// They also need related valid component(s) to be onboarded for account feature to work.
+
+/************
+* Azure tests
+************/
+func TestAccSecureCloudAuthAccountFeature(t *testing.T) {
+	rText := func() string { return acctest.RandStringFromCharSet(10, acctest.CharSetAlphaNum) }
+	accID := rText()
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck: func() {
+			if v := os.Getenv("SYSDIG_SECURE_API_TOKEN"); v == "" {
+				t.Fatal("SYSDIG_SECURE_API_TOKEN must be set for acceptance tests")
+			}
+		},
+		ProviderFactories: map[string]func() (*schema.Provider, error){
+			"sysdig": func() (*schema.Provider, error) {
+				return sysdig.Provider(), nil
+			},
+		},
+		Steps: []resource.TestStep{
+			{
+				Config: secureAzureWithServicePrincipalFeature(accID),
+			},
+			{
+				ResourceName:      "sysdig_secure_cloud_auth_account.azure_sample",
+				ImportState:       true,
+				ImportStateVerify: true,
+			},
+		},
+	})
+}
+
+func secureAzureWithServicePrincipalFeature(accountID string) string {
+	// to replicate user behavior, snippet creates an actual azure account and
```

Comment:
However, this is the context of the TF acceptance tests. We can discuss and clarify offline the contents of TF snippets from the perspective of a user.